### PR TITLE
fix: online debug body params support content-type x-www-form-urlencoded

### LINKED ISSUE #1196: bug: use online debug post a x-www-form-urlencoded data, which result is not what was expected
# Bug report

## How to Reproduce

```
Request URL: http://127.0.0.1:9000/apisix/admin/debug-request-forwarding
Request Method: POST
Reuqest Body: {"url":"http://httpbin.org/post","request_protocol":"http","method":"POST","body_params":{"some":"data"},"header_params":{"Content-Type":["application/x-www-form-urlencoded"]}}
```
Result:
image

Expected result: 
```
{
  "args": {}, 
  "data": "", 
  "files": {}, 
  "form": {
    "some": "data"
  }, 
  "headers": {
    "Accept": "*/*", 
    "Accept-Encoding": "gzip, deflate", 
    "Content-Length": "9", 
    "Content-Type": "application/x-www-form-urlencoded", 
    "Host": "httpbin.org", 
    "User-Agent": "python-requests/2.21.0", 
    "X-Amzn-Trace-Id": "Root=1-5ff2cd52-10f44a401484d9d51360aea1"
  }, 
  "json": null, 
  "origin": "14.116.71.4", 
  "url": "http://httpbin.org/post"
}
```

ping @nic-chen @liuxiran

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix

- Related issues
part of #1196 
fix #1196 
should be merged after #1202

___
### Bugfix
- How to fix?
image

add `radio group` to enable the body params supporting different content-types:

- `none` for `did not have body params`
- `x-www-form-urlencoded` for `content-type:application/x-www-form-urlencoded`
- `json` for `content-type: application/json`
- `raw input` for `other content-type`

TODO：support `content-type: application/form-data`, which is used to upload files.

### CODE REVIEW

File: web/src/pages/Route/components/DebugViews/DebugDrawView.tsx [hunk 1]
```diff
@@ -48,32 +50,55 @@ const DebugDrawView: React.FC<RouteModule.DebugDrawProps> = (props) => {
...
   const methodWithoutBody = ['GET', 'HEAD'];
+  const [bodyCodeMirrorMode, setBodyCodeMirrorMode] = useState(DEBUG_BODY_CODEMIRROR_MODE_SUPPORTED[0].mode)
 
   const resetForms = () => {
     queryForm.setFieldsValue(DEFAULT_DEBUG_PARAM_FORM_DATA);
     bodyForm.setFieldsValue(DEFAULT_DEBUG_PARAM_FORM_DATA);
     headerForm.setFieldsValue(DEFAULT_DEBUG_PARAM_FORM_DATA);
     authForm.setFieldsValue(DEFAULT_DEBUG_AUTH_FORM_DATA);
     setResponseCode(`${formatMessage({ id: 'page.route.debug.showResultAfterSendRequest' })}`);
+    setBodyType('none');
```
Comment (juzhiyuan) [inline web/src/pages/Route/components/DebugViews/DebugDrawView.tsx:64]: would better use `enum` for a clear description

File: web/src/pages/Route/components/DebugViews/DebugDrawView.tsx [hunk 2]
```diff
@@ -233,7 +258,54 @@ const DebugDrawView: React.FC<RouteModule.DebugDrawProps> = (props) => {
...
-                <DebugParamsView form={bodyForm} />
+                <Radio.Group onChange={(e) => {setBodyType(e.target.value)}} value={bodyType}>
+                  <Radio value="none">None</Radio>
+                  {
+                    DEBUG_BODY_TYPE_SUPPORTED.map((type) => (
+                      <Radio value={type} key={type}>{type}</Radio>
+                    ))
+                  }
+                </Radio.Group>
+                {bodyType === 'raw input' && <Select
```
Comment (juzhiyuan) [inline web/src/pages/Route/components/DebugViews/DebugDrawView.tsx:269]: aha, `enum` is useful here :)

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

File: web/src/pages/Route/components/DebugViews/DebugDrawView.tsx (hunk 1 above)
Comment (liuxiran) [reply web/src/pages/Route/components/DebugViews/DebugDrawView.tsx]: good catch~`enum` improved readability of the code :+1: 

fixed, thanks

File: web/src/pages/Route/components/DebugViews/DebugDrawView.tsx (hunk 2 above)
Comment (liuxiran) [reply web/src/pages/Route/components/DebugViews/DebugDrawView.tsx]: fixed

**Comment by juzhiyuan** [comment]
cc @liuxiran

**Comment by liuxiran** [comment]
> cc @liuxiran

updating~

**Pull request merged**